### PR TITLE
[defines] Refuse compilation when both IONIZED and NEUTRAL fluids are…

### PR DESCRIPTION
**Pull request by gawrysz**
… defined

### CODE REVIEW

**Comment by gawrysz** [comment]
Jenkins test this please

File: src/base/defines.c [hunk 1]
```diff
@@ -65,7 +65,10 @@
 #endif
 
 #ifdef NEUTRAL
-#define FLUID
+#  define FLUID
+#  ifdef IONIZED
+#    error Currently there are no solves that can manage a mixture of neutral and ionized fluid
```
Comment (Xarthisius) [inline src/base/defines.c]: s/solves/solvers/

**Comment by gawrysz** [comment]
Jenkins test this please

**Comment by gawrysz** [comment]
Jenkins test this please

**Pull request merged**